teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -131,194 +134,110 @@ surfacing the verification information and building a corpus of verified resourc
-  file in the existing Catalog at `tektoncd/catalog`.
+
+* **Tekton Catalog Maintainers**: The core group of OWNERS who can approve changes in verified Tekton Catalogs.
+Today, they are defined in [OWNERS][catalog-owners] file in the existing Catalog at `tektoncd/catalog`. The current maintainers of `tektoncd/catalog` should become the maintainers in the `tektoncd-catalog` github org.
 
 ## Proposal
 
 ### Ownership and Maintenance
 
-As previously discussed in [TEP-0003][tep-0003-upstream], we propose
-creating two support tiers, `Community` and `Official`, through 
-separate Tekton Catalogs.
+As previously discussed in [TEP-0115][tep-0115-org] and [migration](https://github.com/tektoncd/hub/issues/667) to the Artifact Hub[hub],
+the Tekton Community has decided to migrate to the decentralized model for Catalogs and use the Artifact Hub to surface Tekton Catalogs.
+Given the above changes, we propose creating two support tiers: `Community` and `Verified`. Community Catalogs make it easy for Contributors to share resources, while the Verified Catalogs provide high quality resources that users can rely on. Community and Verified Catalogs will be published in the [Hub][hub].
 
-The Community Catalog makes it easy for contributors to share resources, 
-while the Official Catalog provides a corpus of high quality resources 
-that users can rely on.
+#### Community Catalogs
 
-Both the Community and Official Catalogs will be:
-* Owned by the [Tekton Catalog Maintainers][catalog-owners].
-  The resources in the Community Catalog are owned by contributors,
-  but the Catalog itself is owned by the Tekton Catalog Maintainers.
-* Published in the [Tekton Hub][hub].
-* Share infrastructure, such as testing and scanning tooling. 
+To ensure the workload of maintaining shared resources is sustainable, Contributors can share and maintain resources
+in their own Community Catalogs. Community Catalogs will provide a low barrier of entry, in the testing and security 
+requirements, to encourage community contributions.
 
-#### Tekton Community Catalog
-
-To ensure the workload of maintaining shared resources is sustainable, 
-contributors can share and maintain resources in the Community Catalog.
-The Community Catalog will maintain a low barrier of entry, in the
-testing and security requirements, to encourage community contributions.
+##### Requirements
 
-The current Tekton Catalog at `tektoncd/catalog` will be reused as the 
-Community Catalog.
+1. The Catalog MUST comply with the contract and versioning defined in [TEP-0003][tep-0003-org] or [TEP-0115][tep-0115].
+We will support the directory-based versioning for backwards compatibility during migration; new Catalogs should use the
+git-based versioning defined in TEP-0115.
+2. The Catalog MUST define an OWNER file that specifies at least one Maintainer.
+3. The Catalog MAY have automated testing using Tekton. If there are any failures, they MAY be resolved. The automated 
```

Comment:
for community catalogs, having automated testing using Tekton might not be possible for all community owners. How about we just keep "The Catalog MAY have automated testing enabled." ?